src/Symfony/Component/Uid/Uuid.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Uid;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class Uuid implements \JsonSerializable
```

Comment:
This class should be marked as experimental, now as it’s not part of string component anymore

Reply:
only if the component is made experimental - that's not been discussed yet.

Reply:
Ah ok I thought its part of the process to ease fixing stuff, based on experience